airbyte-integrations/connectors/source-stripe/unit_tests/resource/http/response/external_account_cards.json

```diff
@@ -0,0 +1,34 @@
+{
+  "object": "list",
+  "url": "/v1/accounts/acct_1032D82eZvKYlo2C/external_accounts",
```

Comment:
I don't like the fact that this is not the real URL that is being poked but no behavior rely on this so I don't think it's worth fixing now. The same applies for `external_bank_accounts.json`

Reply:
why can't we have a real request with the actual API? Is it because there's no data in our account?

Reply:
The concern comes from the fact that the id in `"/v1/accounts/<id>/external_accounts"` depends on the parent stream. However, we don't use this field in our logic so there is no need really to make sure the values are align with the parent in the test

Reply:
Could you add a comment explaining that here?

Reply:
This is JSON so I don't think I can add one specifically in this file. As it is only use in `test_external_account_cards`, I'll add that there